client/settings/connect-stripe-account/index.js

```diff
@@ -30,11 +31,19 @@ const TermsOfServiceText = styled.p`
 	margin: 22px 0px 16px;
 `;
 
+const ErrorContainer = styled.div`
+	margin-bottom: 12px;
+`;
+
 const ButtonWrapper = styled.div`
 	align-items: center;
 	display: flex;
 	flex-wrap: wrap;
 
+	> :first-child {
+		margin-right: 8px;
+	}
+
```

Comment:
Nit: I think it would be clearer to use `column-gap` or even `gap` instead so it's a bit clearer what we're aiming for. I also think we should aim for a larger gap by default.
```suggestion
	flex-wrap: wrap;
	column-gap: 12px;

```

Reply:
Changed in [d7d0e06](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4818/commits/d7d0e06112a4b07dd2ff6c2f2d6abc924a7650ac)